### ISSUE TITLE
Over allocatie in WLO

### ISSUE THREAD

**Issue by jipclaassens** [ObjectVision/RSopen#217]
@basvanbemmel @IsmayPBL ik heb jullie scenario gereproduceerd. 

y2030:
![image](https://github.com/user-attachments/assets/f0a9406b-0d58-4fe7-8ddf-620ac72b13b2)

y2040:
![image](https://github.com/user-attachments/assets/5ef5992e-cf30-45be-ad9b-f887b6b01295)

Oftwel, er lijkt iets fout te gaan in 2040. Ik heb dus geen andere parameters aangepast.

**Comment by jipclaassens** [comment]
WIP:

Een eerste bevinding zijn hoge claims:
- y2022: 7.98 miljoen woningen
- y2030: 8.63 miljoen woningen, oftewel +650,000 woningen
- y2040: 9.45 miljoen woningen, oftewel +820,000 woningen

De overheid heeft plannen, maar of het realistisch is?

Ik vermoed dat de eerste 650,000 woningen geplaatst worden door onder andere onderbenutte locaties (lees: lage dichtheden) te herontwikkelen in hogere dichtheden. Hierdoor wordt relatief weinig ruimte ingenomen.

**Comment by jipclaassens** [comment]
NVM afoort:
- totaal y2022 135,487
- claim y2030 149,624
- totaal y2030 s0i0 146,306
- totaal y2030 SnA 149,547
- claim y2040 168,954
- totaal y2040 s0i0 158,905
- totaal y2040 SnA 168,566

**Comment by jipclaassens** [comment]
De vele ontwikkeling in bestaand bebouwd gebied (lees: Bilthoven) is verholpen door de parameter `BouwenInWoonlocatieNietVanGemeenteOfCorporatie` op false te zetten. Deze stond ook zo in de WLO2024 svn repo, maar niet hier in RSo. Nu wel.

**Comment by jipclaassens** [comment]
Hier in Utrecht zie ik deze vlakken MGVS ontstaan in y2040.

![image](https://github.com/user-attachments/assets/9aa554a7-6029-4fef-95f7-bb64856f67d8)

Suitability is identiek in 2030 en 2040: namelijk 5,440,227
Zaaglijn 2030 = 17,031,194
Zaaglijn 2040 = 5,439,935

Oftewel, die locaties waren in 2030 nog niet interessant genoeg, en in 2040 wel.

**Comment by jipclaassens** [comment]
prov utrecht:

StartState, 594,000 woningen

BAG nieuwbouw, ss_rel, 7,000 cellen wonen
BAG nieuwbouw, totaal, 30,759 woningen
= 4,39 woning per cell = 154.52 per ha

y2030, ss_rel, 28,000 cellen wonen
y2030 -> SnA=661,000 woningen
Oftewel, 21,000 gealloceerde wooncellen met daarin 37,000 woningen = 1.76 woning per cel = 28.2 per ha

y2040,s0_nvm_i0 nieuw gealloceerde cellen  ~20,000 
y2040,s0_nvm_i1 nieuw gealloceerde cellen  ~6,882 
y2040,s0_nvm_i2 nieuw gealloceerde cellen  ~6,200
y2040,s1_prov_i0 nieuw gealloceerde cellen  ~2,930
y2040,s1_prov_i1 nieuw gealloceerde cellen  ~1,259
y2040,s1_prov_i2 nieuw gealloceerde cellen  ~625
y2040,s1_nvm_i0 -> SnA=745,000 woningen

Oftewel, 38,000 nieuwe wooncellen, met daarin 84,000 woningen = 2.2 woning per cel = 35.2 per ha

**Comment by basvanbemmel** [comment]
ok dank voor alle info, ik moet het nog even goed lezen hoe of wat, ik had jouw spul gemerged in onze WLO ik krijg nu:
Maar wellicht moet ik even wachten tot alles af is wellicht, laat maar weten

![image](https://github.com/user-attachments/assets/b878efbc-1259-49f9-b9b4-8e0ec9ef68b2)

**Comment by jipclaassens** [comment]
> ok dank voor alle info, ik moet het nog even goed lezen hoe of wat, ik had jouw spul gemerged in onze WLO ik krijg nu: Maar wellicht moet ik even wachten tot alles af is wellicht, laat maar weten

Ja je moet ook nog eventjes de SD updaten.

**Comment by basvanbemmel** [comment]
ha ok dank

**Comment by basvanbemmel** [comment]
na de SD update krijg ik (is het wel de bedoeling dat er al gerund kan worden of ben je eigenlijk nog bezig?)

![image](https://github.com/user-attachments/assets/b56ec30e-b58f-4fe8-8cba-4baebecbdab5)

**Comment by basvanbemmel** [comment]
Er lijkt een probleem te zijn met item
/SourceData/Vastgoed/BAG/PerJaar/Y2022/pand/WP5_rel
WP5_rel.fss bestaat niet en ik zie ook niet waar dit aangemaakt wordt

**Comment by basvanbemmel** [comment]
na  nieuwe merge popt deze melding op ook wel es gehad bij WLO svn. Wat is dat niet iets met verblijfsreceatie?

![image](https://github.com/user-attachments/assets/5460487a-f279-473d-b5d6-20a0ff588df7)

**Comment by basvanbemmel** [comment]
Aangezien op:
https://github.com/ObjectVision/RSopen/commit/29e7744aaca984e9dc0f072e3cee2f11dfb54e56
geen WIP staat heb ik deze gemerged.
Ik krijg  nu (let op ondeze SectorAlloc is bij ons anders, zie onze WLO-branch https://github.com/pbl-nl/model-RSopen/blob/WLO/cfg/main/VariantParameters.dms)


![image](https://github.com/user-attachments/assets/fa10360f-2323-4fe2-a3ae-628d810360a2)

Heeft te maken met regel 77 in DichtheidZichtjaar_T.dms
is gewijzigd door jouw op woensdag de 18e dec. van
					first(CaseClassifications/VariantK/ZonCombinerenMetBestaandGroen) && BaseData/StartState/Grondgebruik/Per_AllocDomain/IsGroen ?

naar
					CaseClassifications/VariantK/ZonCombinerenMetBestaandGroen[/Variant_rel] && BaseData/StartState/Grondgebruik/Per_AllocDomain/IsGroen ?

Als ik dit terug verander krijg ik weer een andere error (integrity check error).

![image](https://github.com/user-attachments/assets/98e04ad9-6fd1-4f3c-b681-106757d20cb0)

en ik verwacht na het oplossen van deze error het 
-1073740940
probleem
Ik meen mij ter herinneren dat dit te maken had met een integrity check op verblijfsrecreatie
Zie versie 11290 van https://pbl.sliksvn.com/ruimtescanner/PBL/ProjDir/branches/WLO2024

          max_elem vanwege integrity check failure
          zie ook https://github.com/ObjectVision/RSopen/issues/174
          Bron RSOpen: 12 november 2024 SHA-1: b08d749d373880bf1ea920ec349e943de446af90
          
          50-> 100 vanwege Drenthe

**Comment by jipclaassens** [comment]
> naar CaseClassifications/VariantK/ZonCombinerenMetBestaandGroen[/Variant_rel] && BaseData/StartState/Grondgebruik/Per_AllocDomain/IsGroen ?

Klopt, dit had ik moeten aanpassen en had met Zon aan niet getest. Nu wel en gecommit, iets anders dan terugzetten. 

> Als ik dit terug verander krijg ik weer een andere error (integrity check error).
> 
> ![image](https://github.com/user-attachments/assets/98e04ad9-6fd1-4f3c-b681-106757d20cb0)
> 
> en ik verwacht na het oplossen van deze error het -1073740940 probleem Ik meen mij ter herinneren dat dit te maken had met een integrity check op verblijfsrecreatie 

Nee dit keer niet. Ik check telkens bij het inlezen van ontkoppelde tifs of niet de hele tif undefined of 0 is (in het verleden hebben we hier nog wel eens problemen mee gehad). Ik vermoed dat het nu geen issue meer is, maar desondanks is het een nuttige check om te houden. Echter, nu we een studygebied hebben midden in het land is er geen OSA (OutsideStudyArea, lees: buitenland) en exteriorwater (e.g. noordzee). Oftewel de ontkoppelde BBG tifs voor studygebied Utrecht falen deze check, want alles is 0. Ik zal deze check losser zetten in het geval van een studyarea keuze.

![Image](https://github.com/user-attachments/assets/7db9435d-a1df-4dfd-82ee-97d680e82b03)

Beide zijn gecommit en het rekent bij mij door.

**Comment by basvanbemmel** [comment]
De gehele batch runt bij mij nu ook goed, dank
